data/src/test/java/org/apache/iceberg/io/TestPositionDeltaWriters.java

```diff
@@ -20,43 +20,42 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
 import java.util.List;
 import org.apache.iceberg.DataFile;
 import org.apache.iceberg.DeleteFile;
 import org.apache.iceberg.FileFormat;
+import org.apache.iceberg.Parameter;
+import org.apache.iceberg.ParameterizedTestExtension;
+import org.apache.iceberg.Parameters;
 import org.apache.iceberg.PartitionSpec;
 import org.apache.iceberg.RowDelta;
 import org.apache.iceberg.expressions.Expressions;
 import org.apache.iceberg.relocated.com.google.common.collect.ImmutableList;
 import org.apache.iceberg.util.StructLikeSet;
 import org.junit.Assert;
```

Comment:
still a JUnit4-Assert left here

Reply:
@nastra Thanks for the review. I only migrate classes related to `TableTestBase` to delete the class, so I keep these classes JUnit4 because there should be a lot of changes if each class is updated to JUnit5. And, I'm thinking that after deleting `TableTestBase`, I'll migrate those classes to Junit 5. Should I update all classes in this PR to JUnit 5? or partially update?

Reply:
yeah indeed, that would lead to many changes. I'm ok doing the conversion from JUnit4 asserts to AssertJ on these files in a separate PR then

Reply:
Thank you. I'll create a PR to migrate them in a separate PR. At least, through this issue, we can delete the `TableTestBase`.